contracts/test/_lib/lib.js

```diff
@@ -1,1472 +1,65 @@
-  assert.equal(event.discprovId, id, 'Expected same discprov id')
-  assert.equal(event.discprovWallet, wallet, 'Expected same discprov wallet address')
-  assert.equal(event.discprovEndpoint, endpoint, 'Expected same discprov endpoint')
-
-  return event
-}
-
-/** Validate that discprov fields matches inputs */
-export const validateRegisteredDiscprov = (discprov, wallet, endpoint) => {
-  assert.equal(discprov.wallet, wallet, 'Expected same discprov wallet address')
-  assert.equal(discprov.endpoint, endpoint, 'Expected same discprov endpoint')
-}
-
-/** Remove contract from registry and confirm contract instance is dead */
-export const unregisterContractAndValidate = async (registry, contractRegistryKey, contractInstanceAddress) => {
-  await registry.removeContract(contractRegistryKey)
-  await assertNoContractExists(contractInstanceAddress)
-}
-
-/** Customizable console.log wrapper */
-export const deepLog = (msg, val, depth = null, showHidden = false) => {
-  console.log('\n-- ' + msg + ' --\n', util.inspect(val, { colors: true, depth: depth, showHidden: showHidden }), '\n')
-}
-
-/** Returns decoded multihash with digest, hash function, and digest size
- *  IPFS Uses alphanumeric base58 encoded 46byte hash which cannot be stored by default in solidity bytes32 field
- *  https://github.com/multiformats/multihash + https://github.com/saurfang/ipfs-multihash-on-solidity
- *  @param {string} - encoded multihash - 46byte long base58-encoded string
- *  @returns {object} with:
- *    digest - 32byte long hex string
- *    hashFn - 1byte int indicating hash function used
- *    size - 1byte int indicating size of digest
- */
-export const decodeMultihash = (multihash) => {
-  let base16multihash = bs58.decode(multihash)
-  return {
-    digest: `0x${base16multihash.slice(2).toString('hex')}`,
-    hashFn: base16multihash[0],
-    size: base16multihash[1]
-  }
-}
-
-/** Returns encoded multihash given component object
- *  @param {string, string, string} - digest, hash function, size
- *  @returns {string} - base58-encoded 46byte string
- */
-export const encodeMultihash = (digest, hashFn, size) => {
-  let hashBytes = Buffer.from(digest.slice(2), 'hex')
-  let multiHashBytes = Buffer.from(new ArrayBuffer(2 + hashBytes.length))
-  multiHashBytes[0] = hashFn
-  multiHashBytes[1] = size
-  multiHashBytes.set(hashBytes, 2)
-  return bs58.encode(multiHashBytes)
-}
-
-/** Asserts that no contract exists at the given address.
- *  @param {string} - address to check
- */
-export const assertNoContractExists = async (contractAddress) => {
-  const contractCode = await web3New.eth.getCode(contractAddress)
-  assert(
-    contractCode === '0x0' || contractCode === '0x', // geth returns 0 as '0x', ganache returns it as '0'. This supports both.
-    'Expected no contract at given address')
+/** Wrapper class for all e2e tests */
+
+import { unregisterContractAndValidate, assertNoContractExists } from './testContract'
+import {
```

Comment:
i don't believe you need to import each property and expose these one by one here. You should be able to do something like


js```
import * as testContract from './testContract'
import * as testPlaylist from './testPlaylist'
...

export { ...testContract, ...testPlaylist, ...<otherFile> }

```

Reply:
👍